src/styles/card.scss

```diff
@@ -0,0 +1,124 @@
+@import '../styles/variables';
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  max-width: $card-width;
+  border-radius: $border-radius;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    width: calc($card-width - 40px);
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), avoid using `calc` unless absolutely necessary. If `$card-width` is a fixed value and the subtraction is also a constant, consider precalculating this value and storing it in a variable.